jsonrpc/web3_endpoint.go

```diff
@@ -13,15 +14,27 @@ type Web3 struct {
 	chainName string
 }
 
-var clientVersionTemplate = "%s [chain-id: %d] [version: %s]"
+var clientVersionTemplate = "%s-%d/%s/%s-%s/%s"
 
 // ClientVersion returns the version of the web3 client (web3_clientVersion)
+// Example: "polygon-edge-53105/v1.1.0/linux-amd64/go1.20.0"
+// Spec: https://ethereum.org/en/developers/docs/apis/json-rpc/#web3_clientversion
 func (w *Web3) ClientVersion() (interface{}, error) {
+	var version string
+	if versioning.Version != "" {
+		version = versioning.Version
+	} else if versioning.Commit != "" {
+		version = versioning.Commit[:8]
+	}
+
 	return fmt.Sprintf(
 		clientVersionTemplate,
 		w.chainName,
 		w.chainID,
```

Comment:
It doesn't seem that any other Ethereum implementation embeds `chain id` into the return value, so I'd probably remove it.

Reply:
The official spec says that the endpoint should return a client version. It does not say what specific information it should return so I guess it is up to devs. Currently, this string includes a chain ID so just decided to keep it in a new format as well. Do you have any doubts why the chain ID should be removed from there?

Reply:
Ok, done